### ISSUE TITLE
Blank MM display

### ISSUE THREAD

**Issue by rroach3753** [seeshaughnessy/MMM-OneTracker#6]
Fresh install of MM and MMPM.  Installed OneTracker (one of my favs).  MM works fine until I added my creds to the OneTracker config.  As soon as I restart MM after this, it's just a blank screen.  And MM no longer works going forward, even after removing my creds.

**Comment by rroach3753** [comment]
Hello?

**Comment by seeshaughnessy** [comment]
To further troubleshoot, please take a look at this documentation, and try to provide some additional details that appear on the debug or log files. The error that you mention can be caused by several different things, and I'm not sure where to start troubleshooting: https://gist.github.com/lavolp3/db7de8c0664ba95e8b9247ba685095fe

**Comment by spyder007** [comment]
I'm getting the same issue:  it would seem that this module is trying to use the request module, which was deprecated in February and removed from Magic Mirror's core.

https://github.com/MichMich/MagicMirror/issues/2468

I am interested in this module, if I get some time I may fork your repository and see if I can change it.

**Comment by rroach3753** [comment]
Yeah, I noticed it didn't start happening until after the last MagicMirror update I did (about a month or so ago).  Oddly, if you have the module installed and add the code but not restart, it sometimes will work...but once you restart MM it breaks.  It's not just this module either.  It's also doing it with the MMM-SystemStats module.

**Comment by spyder007** [comment]
See pr #7 for a fix.

**Comment by rroach3753** [comment]
Working perfectly now.

**Comment by rroach3753** [comment]
Now if there was a way to not hide the module when no packages were being delivered :)

**Comment by jdlucke** [comment]
following the thread, if the fix spyder007 mentions above works, how do I pull that in.  Do I just need to overlay the code that I have for the module ?  has it been fixed in the main branch?  I have other modules that seem to have broken that are similar with request being noted.

**Comment by rroach3753** [comment]
@seeshaughnessy This issue has returned after updating to the latest MagicMirror version.

**Comment by seeshaughnessy** [comment]
Hey there,
  

  
I made this for my personal magic mirror, which I unfortunately no longer have. I’m not sure I’ll be able to update this going forward. I hope someone else can take over development!
  
  
  
  
  
  
  

  
  
>

**Comment by rroach3753** [comment]
> Hey there, I made this for my personal magic mirror, which I unfortunately no longer have. I’m not sure I’ll be able to update this going forward. I hope someone else can take over development!

This is a sad development!  It's a great module!  Any way I can convince you to look at it?  I'll provide any logs you ask for and test any fixes.  I don't know JS so I can't figure it out myself.
